gcc/d/gdc.texi

```diff
@@ -120,196 +120,662 @@ and the Info entries for @file{gdc} and @file{gcc}.
+@cindex directory options
+@cindex options, directory search
+@cindex search path
 
-@item -fmake-mdeps=@var{filename}
-@cindex @option{-fmake-mdeps}
-Like -fmake-deps=@var{filename} but ignore system header files.
+These options specify directories to search for files, libraries, and
+other parts of the compiler:
 
-@item -fonly=@var{filename}
-@cindex @option{-fonly}
-Process all modules specified on the command line,
-but only generate code for the module specified by the argument.
+@table @gcctabopt
 
-@item -fversion=@var{opt}
-@cindex @option{-fversion}
-Compile in version code into the program.
+@item -I@var{dir}
+@cindex @option{-I}
+Specify a directory to use when searching for imported modules at
+compile time.  Multiple @option{-I} options can be used, and the
+paths are search in the same order.
+
+@item -J@var{dir}
+@cindex @option{-J}
+Specify a directory to use when searching for files in string imports
+at compile time.  This switch is required in order to use
+@code{import(file)} expressions.  Multiple @option{-J} options can be
+used, and the paths are searched in the same order.
+
+@item -L@var{dir}
+@cindex @option{-L}
+When linking, specify a library search directory, as with @command{gcc}.
+
+@item -B@var{dir}
+@cindex @option{-B}
+This option specifies where to find the executables, libraries,
+source files, and data files of the compiler itself.
+
+@item -fmodule-filepath=@var{module}=@var{spec}
+@cindex @option{-fmodule-filepath}
+This option manipulates file paths of imported modules, such that if an
+imported module matches all or the leftmost part of @var{module}, the file
+path in @var{spec} is used as the location to search for D sources.
+This is used when the source file path and names are not the same as the
+package and module hierachy.
+
+For example, compiling with the following @option{-fmodule-filepath}
+options.
+
+@smallexample
+gdc test.d -fmodule-filepath=A.B=foo.d -fmodule-filepath=C=bar
+@end smallexample
+
+Will tell the compiler to search in all import paths for the source file
+@var{foo.d} when importing @var{A.B}, and the directory @var{bar/} when
+importing @var{C}.
+
+As annotated in this D source file:
+
+@smallexample
+import A.B;     // Matches A.B, searches for foo.d
+import C.D.E;   // Matches C, searches for bar/D/E.d
+import A.B.C;   // No match, searches for A/B/C.d
+@end smallexample
+
+@item -imultilib @var{dir}
+@cindex @option{-imultilib}
+Use @var{dir} as a subdirectory of the directory containing target-specific
+D sources and interfaces.
+
+@item -iprefix @var{prefix}
+@cindex @option{-iprefix}
+Specify @var{prefix} as the prefix for subsequent @option{-iwithprefix}
+options. If the @var{prefix} represents a directory, you should include
+the final @code{'/'}.
+
+@item -nostdinc
+@cindex @option{-nostdinc}
+Do not search the standard system directories for D source and interface
+files.  Only the directories you have specified with @option{-I} options
```

Comment:
> directories **you** have

I remember Andrei usually complains about using **you** in documentation. But I don't know if there's any common convention about this and personally I think addressing the reader like this is OK.